mmdet3d/models/losses/centerpoint_focal_loss.py

```diff
@@ -0,0 +1,48 @@
+import torch
+from torch import nn as nn
+
+from mmdet.models.builder import LOSSES
+
+
+@LOSSES.register_module()
+class CenterPointFocalLoss(nn.Module):
```

Comment:
This loss is already implemented as [GaussianFocalLoss in mmdet](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/gaussian_focal_loss.py#L30)